pkg/http/server.go

```diff
@@ -107,7 +107,7 @@ func (s *server) StartNonBlocking() error {
 		listeners = append(listeners, l)
 	} else {
 		for _, apiListenAddress := range s.config.APIListenAddresses {
-			addr := net.JoinHostPort(apiListenAddress, strconv.Itoa(s.config.Port))
+			addr := apiListenAddress + ":" + strconv.Itoa(s.config.Port)
```

Comment:
why is net.JoinHostPort removed? can there be IPv6 scenarios where that method would be helpful?

Reply:
See here: https://github.com/dapr/dapr/pull/5648#discussion_r1053738839